doc/supportedPlatformVersions.md

```diff
@@ -398,6 +398,9 @@
 - 16.13.2
 - 16.14.0
 - 16.14.2
+- 18.0.0
+- 18.1.0
+- 18.2.0
```

Comment:
unrelated to this change, looks like node 18 was incorrectly placed under the stretch versions and running the generate script fixed it